include/gemmini.h

```diff
@@ -767,7 +642,7 @@ static void matmul_cpu(size_t dim_I, size_t dim_J, size_t dim_K,
         result = result < 0 ? 0 : (result > max ? max : result);
       }
 
-      C[i][j] = (elem_t)result;
+      *((elem_t*)C + i*stride_C + j) = (elem_t)result;
     }
   }
 }
```

Comment:
Nit: Remove the commented `matmul_cpu`?

Reply:
@hngenc , do you need this for any reason?

Reply:
Keep the comments for now, please